### ISSUE TITLE
Plan to do it for Stunts ?

### ISSUE THREAD

**Issue by naworm** [whampson/pigeon-locator#8]
Plan to do it for Stunts ?

**Comment by whampson** [comment]
Yeah I might, I've been thinking of expanding this into a basic save editor so I'll probably include Stunts with that.

**Comment by naworm** [comment]
> Yeah I might, I've been thinking of expanding this into a basic save editor so I'll probably include Stunts with that.

Could be great, I missed one stunt and now I have to redo all of them 😄
